client/src/components/layout/BuildForm.js

```diff
@@ -93,26 +92,29 @@ const BuildForm = (props) => {
                         const serverData = await newBuildFormResponse.json()
                         const serverErrors = translateServerErrors(serverData.errors)
                         return setBuildFormErrors(serverErrors)
-                    }
-                    else {
+                    } else {
                         const errorMessage = `${newBuildFormResponse.status} - ${newBuildFormResponse.statusText}`
                         const error = new Error(errorMessage)
                         throw (error)
                     }
                 }
                 else {
```

Comment:
should be "} else {" to follow pattern

Reply:
Edited in https://github.com/MarlonS419/better-builds/pull/7/commits/96a8f114b6ad6ea080b81e63593f7408da0d9aea